src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMEnvironmentContributor.java

```diff
@@ -0,0 +1,89 @@
+package com.atlassian.bitbucket.jenkins.internal.scm;
+
+import com.google.common.annotations.VisibleForTesting;
+import edu.umd.cs.findbugs.annotations.NonNull;
```

Comment:
please don't import `NonNull` it is not needed. We have package level annotations that means that every argument and every return type is not null, unless annotated to say it can be null. 

Can I also ask you to add this to the checkstyle as a banned import? We already have the `javax.annotation.Nonnull` and given you landed on this one we should help future selves to not commit it :D